app/components/pay-page/choose-membership-plan-modal/plan-card.hbs

```diff
@@ -0,0 +1,100 @@
+<div
+  class="flex items-start justify-between border
+    {{if @isSelected 'border-teal-500' 'border-gray-200 hover:border-gray-300'}}
+    rounded p-3
+    {{if @isSelected 'bg-teal-50' 'bg-white hover:bg-gray-50'}}"
+  role="button"
+  {{on "click" this.handleClick}}
+  data-test-plan-card
+  ...attributes
+>
+  <div class="flex items-start gap-2">
+    <div
+      class="flex-shrink-0 flex items-center justify-center w-4 h-4 border rounded-full mt-0.5 {{if @isSelected 'border-teal-500' 'border-gray-400'}}"
+    >
+      {{#if @isSelected}}
+        <div class="w-2.5 h-2.5 bg-teal-500 rounded-full"></div>
+      {{/if}}
+    </div>
+
+    <div>
+      <div class="flex items-center gap-x-1.5">
+        <span class="text-gray-600 font-bold text-sm">
+          {{@plan.title}}
+          pass
+        </span>
+        {{#if (eq @plan.id "yearly")}}
+          <span class="inline-block bg-teal-500 text-white font-bold rounded px-1.5 py-1 text-xxs uppercase">best value</span>
+        {{/if}}
+      </div>
+
+      <div class="prose prose-xs prose-compact mt-1 text-gray-500">
+        {{#if @plan.validityInMonths}}
+          <p>
+            Access to all membership benefits for
+            {{@plan.title}}
+          </p>
+        {{else}}
+          <p>
+            Access to all membership benefits forever
+          </p>
+        {{/if}}
+      </div>
+
+      {{#if this.discountedPriceInDollars}}
+        <div class="prose prose-xs prose-compact text-gray-500">
+          <ul class="mt-0">
+            {{#if @regionalDiscount}}
+              <li data-test-regional-discount-notice>
+                <b class="text-teal-500">{{@regionalDiscount.percentOff}}% off</b>
+                —
+                {{@regionalDiscount.countryName}}
+                discount
+              </li>
+            {{/if}}
+            {{#if @promotionalDiscount}}
+              <li data-test-promotional-discount-notice>
+                <b class="text-teal-500">{{@promotionalDiscount.percentageOff}}% off</b>
+                —
+                {{#if @promotionalDiscount.isFromAffiliateReferral}}
+                  <span class="text-nowrap">{{@promotionalDiscount.affiliateReferral.affiliateLink.usernameForDisplay}}'s</span>
+                  referral offer, expires in
+                  <span class="font-mono">{{this.promotionalDiscountCountdownTimerText}}</span>
+                {{else if @promotionalDiscount.isFromStage2Completion}}
+                  Stage 2 completion discount, expires in
+                  <span class="font-mono">{{this.promotionalDiscountCountdownTimerText}}</span>
+                {{else if @promotionalDiscount.isFromSignup}}
+                  Signup discount, expires in
```

Comment:
Could move the whole expiry thing out here and place it below: 

```ts
, expires in <font-mono>....</blah>
```

This is similar to how the first portion above is constant too (`percentOff% -`)